modules/apps/29-fee/client/cli/query.go

```diff
@@ -149,3 +149,46 @@ func GetCmdTotalTimeoutFees() *cobra.Command {
 
 	return cmd
 }
+
+// GetIncentivizedPacketByChannel returns all of the unrelayed incentivized packets on a given channel
+func GetIncentivizedPacketByChannel() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "packets-by-channel [port-id] [channel-id]",
```

Comment:
Feel free to ignore... mega nit 

```suggestion
		Use:     "packets-for-channel [port-id] [channel-id]",
```